docs/data/versions.json

```
@@ -1,11 +1,12 @@
 {
 	"params": {
-		"currentVersion": "v1",
+		"currentVersion": "v2",
 		"ltsVersion": "v1",
-		"previousVersions": []
+		"previousVersions": ["v1"]
 	},
 	"apiDocsDirectory": "/docs/api/",
 	"apiVersionOptionName": {
-		"Fluid Framework 1.0": "v1"
+		"Fluid Framework 1.0": "v1",
+		"Fluid Framework 2.0 Alpha": "v2"
```

We're calling our release candidate a "BETA". So I would recommend:
```suggestion
		"Fluid Framework 2.0 (BETA)": "v2"
```